### PR TITLE
Update debugging.md

### PR DESCRIPTION
**Pull request by dvhirst**
Reading for content; editing for consistency.

### CODE REVIEW

File: docs/tools/debugging.md [hunk 1]
```diff
@@ -327,7 +262,7 @@ Your next step is not to fix the error, but to first give yourself enough inform
 
 **Viewing errors in logfiles**
 
-The webserver can be configured to display errors to screen, but it also logs errors to files on disk. These files vary depending on your hosting environment, so you might consult your webhost's documentation to locate them. You might look for errors in some of these locations depending on webserver/php config:
+The webserver can be configured to display errors to screen, but it also logs errors to files on disk. These files vary depending on your hosting environment, so you might consult your webhost's documentation to locate them. You might look for  errors in some of these locations depending on webserver/php config:
```
Comment (seamuslee001) [inline docs/tools/debugging.md]: @dvhirst looks like there is an extra space came in next to errors

**Comment by seamuslee001** [comment]
Changes look ok, made 1 comment which should be resolved prior to merging

**Comment by dvhirst** [comment]
Seamus, I PRed a revised patch-15 branch a few minutes ago.  Also have been in contact with Sean about the whole editing process.  See comments in CiviCRM chat.  Thanks!  DVH

**Comment by seanmadsen** [comment]
@dvhirst glad to see you figured out how to push more commits to a branch. Exciting! :grin:

**Comment by dvhirst** [comment]
Working on local edit setup now.  Thanks for the guidance.

**Pull request merged**